res/css/views/elements/_ImageView.pcss

```diff
@@ -82,6 +82,9 @@ $button-gap: 24px;
 .mx_ImageView_title {
     color: $lightbox-fg-color;
     font-size: $font-12px;
+    position: absolute;
```

Comment:
Using absolute positioning removes the title element from the surrounding document flow, which means that it can now overlap with the user and toolbar elements in the lightbox if the window is narrow.

Instead, I would suggest using Flexbox to solve this problem: if you apply something like `flex-grow: 1; flex-basis: 0` to each of the user, title, and toolbar elements, then they will each share exactly 1/3 of the width when the window is wide, and the user and title elements can still shrink to form a non-centered layout when the window is narrow, and avoid overlapping each other.

Reply:
Got it, will modify it asap

Reply:
So I make a whole new pull request?

Reply:
No need to, you can simply push more commits to this branch and they'll become part of the pull request